### ISSUE TITLE
[v3] Implement new Concept Exercise: tuples

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/fsharp#863]
This issue describes how to implement the `tuples` concept exercise for the F# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of Tuples in F#.

## Learning objectives

- Know what a tuple is.
- Know when tuples should be used.
- Know how to define tuples of different sizes.
- Know that tuples have structural equality.
- Know how to deconstruct tuples (including as parameters).
- Know how to pattern match on tuples.
- Know that tuples are immutable.

## Out of scope

- `struct` tuples.
- Memory and performance characteristics.

## Concepts

The Concepts this exercise unlocks are:

- `tuples`: know what a tuple is; know when tuples should be used; know how to define tuples of different sizes; know that tuples have structural equality; know how to deconstruct tuples; know how to pattern match on tuples; know that tuples are immutable.

## Prerequisites

This exercise's prerequisites Concepts are:

- `pattern-matching`: know how to do pattern matching.
- `basics`: know how to define bindings and functions.

Any data types used in this exercise (e.g. `strings`) should also be added as prerequisites.

### Hints

- [Tuples][tuples]: introduction to tuples.

### After

- [Tuples][tuples]: introduction to tuples.
- [Tuples introduction][tuples-introduction]: another tuple introduction, but digs a bit deeper.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/fsharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[how-to-implement-a-concept-exercise]: https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md
[implemented-exercises]: https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/README.md#implemented-exercises
[reference]: https://github.com/exercism/v3/blob/master/reference/types/tuple.md
[analyzer]: https://github.com/exercism/fsharp-analyzer
[representer]: https://github.com/exercism/fsharp-representer
[meta-design]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/design.md
[meta-config.json]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/config.json
[docs-concept-exercises]: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md
[tuples]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/tuples
[tuples-introduction]: https://fsharpforfunandprofit.com/posts/tuples/

**Comment by Grenkin1988** [comment]
I'd like to try.
Do we need to migrate from previous, or write from scratch?

**Comment by ErikSchierboom** [comment]
Oh and thanks for wanting to work on this!
